ivy/functional/backends/numpy/experimental/elementwise.py

```diff
@@ -268,7 +268,7 @@ def conj(
     out: Optional[np.ndarray] = None,
 ) -> np.ndarray:
     ret = np.conj(x, out=out)
-    if x.dtype == np.bool:
+    if x.dtype == bool:
```

Comment:
Could you please explain why you changed this and how this fixes the test?

Reply:
numpy deprecated the np.bool and replaced it with python built in bool. that is why the test was failing.

Reply:
Thanks for the explanation!